qml/TabLayout.qml

```diff
@@ -248,37 +257,149 @@ Item {
                                 anchors.verticalCenter: parent.verticalCenter
                                 text: "Domain View"
                                 onClicked: {
-                                    tabLayout.tab_model_[current_]["title"]="Domain View"
-                                    if (stack.deep > 1)
+                                    if (mainApplicationView.monitors == 0)
+                                    {
+                                        dialogInitMonitor.open()
+                                    }
+                                    else if (mainApplicationView.monitors == 1)
                                     {
-                                        stack.pop()
+                                        controller.update_available_entity_ids("Domain", "getDataDialogSourceEntityId")
+                                        open_domain_view_(
+                                            tabLayout.tab_model_[current_]["stack_id"],
+                                            entityModelFirst.get(0).id,
+                                            entityModelFirst.get(0).name)
+                                    }
+                                    else
+                                    {
+                                        domain_id_dialog.open()
                                     }
-                                    stack.push(domainViewLayout)
-                                    refresh_layout(current_)
                                 }
                             }
                         }
                     }
                 }
+
+                Component {
+                    id: domainGraphLayout_component
+
+                    DomainGraphLayout
+                    {
+                        id: domainGraphLayout
+                        component_id: stack.stack_id
+
+                        onUpdate_tab_name: {
+                            tabLayout.tab_model_[current_]["title"] = new_name
+                            refresh_layout(current_)
+                        }
```

Comment:
`refresh_layout()` call doesn't update left panel's info. Argument already matches `current_` and there is no call to `controller.domain_click()` function.

Reply:
Nice catch!